src/Network/Network.Test/ScenarioTests/ExpressRouteCircuitTests.cs

```diff
@@ -97,5 +97,13 @@ public void TestExpressRouteCircuitPeeringWithRouteFilter()
         {
             TestRunner.RunTestScript("Test-ExpressRouteCircuitPeeringWithRouteFilter");
         }
-    }
+
+	    [Fact(Skip = "Rerecord tests")]
```

Comment:
Can you re-record test without skipping? And/or sync up with @srividyathirumala what to do with that?

Reply:
Removed the skip. @number213 had recorded yesterday.